lib/ReportHistory.js

```diff
@@ -52,14 +57,21 @@ class ReportHistory {
      * @returns {ReportHistory}
      */
     store(report) {
-        lastReportOffset++;
-
         if (!fs.existsSync(this.storageDirectory)) {
             fs.mkdirSync(this.storageDirectory);
         }
 
         fs.writeFileSync(this.newReportFile, JSON.stringify(report, null, 4));
 
+        const files = fs.readdirSync(this.storageDirectory)
```

Comment:
Looks like we're using this twice. Might be worth moving to another method/getter called something like `storedReports`

Reply:
Done ;)